tests/app/views/test_buyers.py

```diff
@@ -4,9 +4,17 @@
 from ...helpers import BaseApplicationTest
 from dmapiclient import api_stubs, HTTPError
 from dmcontent.content_loader import ContentLoader
+from dmcontent.questions import Question
 import mock
 from lxml import html
 import pytest
+from ...features import disable_feature
+
+from app.buyers.views import buyers
+from dmcontent.content_loader import ContentLoader
+from dmapiclient import DataAPIClient
+import functools
+from string import ascii_uppercase
```

Comment:
ah, i think i removed the bit using that, this is why we should be using flake8 instead of pep8, it complains about unused imports